parsl/app/app.py

```diff
@@ -58,8 +58,8 @@ def __init__(self, func, data_flow_kernel=None, executors='all', cache=False, ig
             self.kwargs['stderr'] = params['stderr'].default
         if 'walltime' in params:
             self.kwargs['walltime'] = params['walltime'].default
-        if 'parsl_resource_specification' in params:
-            self.kwargs['parsl_resource_specification'] = params['parsl_resource_specification'].default
+        if 'parsl_call_specs' in params:
```

Comment:
don't change core parsl! this is a breaking change for all users, and would need some more serious justificiation

Reply:
Change reverted. It's just that the variable name can be better so more information can flow from top down.